webpack.config.js

```diff
@@ -5,7 +5,7 @@ var webpack = require('webpack');
 
 var base = {
     devServer: {
-        contentBase: path.resolve(__dirname, 'playground'),
+        contentBase: false,
```

Comment:
It looks like this change makes us need to navigate to /playground when running the dev server (since the content base is no longer `playground`). Was that intentional?